test/csuite/wt7989_compact_checkpoint/main.c

```diff
@@ -325,6 +290,23 @@ thread_func_checkpoint(void *arg)
     return (NULL);
 }
 
+/*
+ * thread_wait --
+ *     Loop to constantly yield the calling thread until all threads are ready.
+ */
+static void 
+thread_wait(void){
```

Comment:
Invalid location of bracket. Make sure you run `s_all` script.

Reply:
Fixed.